CRM/Member/Form/Membership.php

```diff
@@ -134,11 +134,11 @@ protected function setEntityFields() {
    */
   public function setDeleteMessage() {
     $this->deleteMessage = '<span class="font-red bold">'
-      . ts('WARNING: Deleting this membership will also delete any related payment (contribution) records.' . ts('This action cannot be undone.')
+      . ts('WARNING: Deleting this membership will also delete any related payment (contribution) records.') . ts('This action cannot be undone.')
```

Comment:
Since updating this anyway can you add a space character between these two? It got lost when this was converted to a php string from when it used to be [inside a tpl](https://github.com/civicrm/civicrm-core/commit/59c798c98d2539714f46c30a2c3d81690f6d6051#diff-58e6354a897382eda0b1d9c6732c07705b37fcf2941b579d4842e8114ab0d046L82).

It might even make sense to make this whole paragraph all one string so the translator has more context, but it was like this before too so 50/50.

Reply:
Sorry by space character I meant a literal ' ', like `ts('WARNING: Deleting this membership will also delete any related payment (contribution) records.') . ' ' . ts('This action cannot be undone.')`. Otherwise the two sentences have no space between them.

Reply:
Yes I understood it later. Space added.